### ISSUE TITLE
How to use this module?

### ISSUE THREAD

**Issue by ashokkumar88** [WCChimiiz/react-native-folio-reader#1]

**Comment by Vandanabisht17** [comment]
@ashokkumar88 Did you find out how to use this module?

**Comment by ashokkumar88** [comment]
@Vandanabisht17 Yes. You have to add your `@ReactMethod` in `RNReactNativeFolioReaderModule.java`. You can find the configurations at https://github.com/FolioReader/FolioReader-Android/wiki/Configuration

**Comment by Vandanabisht17** [comment]
@ashokkumar88  actually i am new to react native i am getting confused can you please help me out and show me how you used this module in your react-native app.

**Comment by ashokkumar88** [comment]
@Vandanabisht17 Download the modified module here https://drive.google.com/file/d/1p-r1T5dc76M-u5lAAGHxT-_hi5BJIyjw/view?usp=sharing. 

In the react native component first import the module.
`import RNReactNativeFolioReader from 'wcchimiiz-react-native-folio-reader';`

Then open epub like this
`RNReactNativeFolioReader.show(epubPath,bookid,null,(result)=>{});`

**Comment by Vandanabisht17** [comment]
@ashokkumar88  thank you so much.. I'll try using it now
